Lib/test/test_statistics.py

```diff
@@ -1004,6 +1004,10 @@ def test_nan(self):
             x = statistics._convert(nan, type(nan))
             self.assertTrue(_nan_equal(x, nan))
 
+    def test_raise_type_error(self):
```

Comment:
Let's rename this `test_invalid_input_type()`

Reply:
Done.